src/java/org/apache/cassandra/cql3/constraints/ColumnConstraintScalar.java

```diff
@@ -0,0 +1,167 @@
+        }
+    }
+
+    public ColumnConstraintScalar(ColumnIdentifier param, Operator relationType, String term)
+    {
+        this.param = param;
+        this.relationType = relationType;
+        this.term = term;
+    }
+
+    @Override
+    public void evaluate(Class<? extends AbstractType> valueType, Object columnValue)
+    {
+        Number columnValueNumber;
+        float sizeConstraint;
+
+        try
+        {
+            columnValueNumber = (Number) columnValue;
+            sizeConstraint = Float.parseFloat(term);
+        }
+        catch (NumberFormatException exception)
+        {
+            throw new ConstraintViolationException(param + " and " + term + " need to be numbers.");
+        }
+
+        switch (relationType)
+        {
+            case EQ:
+                if (Float.compare(columnValueNumber.floatValue(), sizeConstraint) != 0)
+                    throw new ConstraintViolationException(param + " value should be exactly " + sizeConstraint);
+                break;
+            case NEQ:
+                if (Double.compare(columnValueNumber.floatValue(), sizeConstraint) == 0)
+                    throw new ConstraintViolationException(param + " value should be different from " + sizeConstraint);
+                break;
+            case GT:
+                if (columnValueNumber.floatValue() <= sizeConstraint)
+                    throw new ConstraintViolationException(param + " value should be larger than " + sizeConstraint);
+                break;
+            case LT:
+                if (columnValueNumber.floatValue() >= sizeConstraint)
+                    throw new ConstraintViolationException(param + " value should be smaller than " + sizeConstraint);
+                break;
+            case GTE:
+                if (columnValueNumber.floatValue() < sizeConstraint)
+                    throw new ConstraintViolationException(param + " value should be larger or equal than " + sizeConstraint);
+                break;
+            case LTE:
+                if (columnValueNumber.floatValue() > sizeConstraint)
+                    throw new ConstraintViolationException(param + " value should be smaller or equal than " + sizeConstraint);
+                break;
+            default:
+                throw new ConstraintViolationException("Invalid relation type: " + relationType);
+        }
+    }
+
+    @Override
+    public void validate(ColumnMetadata columnMetadata) throws InvalidConstraintDefinitionException
+    {
+        if (!(columnMetadata.type instanceof org.apache.cassandra.db.marshal.NumberType))
+            throw new InvalidConstraintDefinitionException(param + " is not a number");
+    }
+
+    @Override
+    public String toString()
+    {
+        return param + " " + relationType + " " + term;
+    }
+
+    @Override
+    public IVersionedSerializer<ColumnConstraint> serializer()
+    {
+        return serializer;
+    }
+
+    @Override
+    public void appendCqlTo(CqlBuilder builder) {
```

Comment:
brace on new line, also please try to go over the patch and check if braces are needed at all, I see a lot of single statements e.g. in for loops etc or in ifs where braces are not necessary at all and that is what our code style is too.

Reply:
Good catch! I'll go through them and take out as much as I can. Also, I will check if this is something we can add to the actual checkstyle. It is way too easy to overlook one right now.